specification/compiler/index.ts

```diff
@@ -35,6 +36,7 @@ compiler
   .step(validateModel)
```

Comment:
LGTM small nitpick: it'd be nicer if `logErrors()` is included in a new method on compiler: `writeErrors()` so that the emit of errors is all one in place.

Reply:
Yes! More or less the same I said in https://github.com/elastic/elastic-client-generator/pull/202#pullrequestreview-612341955 :)